cmd/operator/kodata/knative-serving/0.17.3/2-serving-core.yaml

```diff
@@ -1138,7 +1138,7 @@ data:
     logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
 
     # If true, the request logging will be enabled.
-    # NB: up to and including Knative version 0.18 if logging.requst-log-template is non-empty, this value
+    # NB: up to and including Knative version 0.18 if logging.request-log-template is non-empty, this value
```

Comment:
Ack with @antoineco's comments. These are files pulled in from other upstream projects. Please undo all changes to `cmd/operator/kodata`